TWLight/resources/management/commands/meta_translations.py

```diff
@@ -0,0 +1,62 @@
+import ast
+import logging
+import requests
+from bs4 import BeautifulSoup
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from TWLight.resources.models import Partner
+
+class Command(BaseCommand):
+    help = 'Grabs short and long description translations from Meta and inserts them into the database'
+    
+    def handle(self, *args, **kwargs):
+        all_partners = Partner.even_not_available.all()
+        languages_on_file = settings.LANGUAGES
+
+        languages_on_revision_field = {}
+        for each_partner in all_partners:
+            if each_partner.short_description_last_revision_ids is not None:
+                languages_on_revision_field = ast.literal_eval(each_partner.short_description_last_revision_ids)
+            for every_language in languages_on_file:
+                if every_language[0] not in languages_on_revision_field:
+                    languages_on_revision_field[every_language[0]] = None
+            each_partner.short_description_last_revision_ids = languages_on_revision_field
+
+            response = requests.get('https://meta.wikimedia.org/w/api.php?action=query&format=json&meta=messagegroupstats&mgsgroup=page-Library_Card_platform%2FTranslation%2FPartners%2FShort_description%2F{partner_pk}'.format(partner_pk=each_partner.id))
+            language_data = response.json()
+            if 'error' in language_data:
+                print 'No short_description for partner {}.'.format(each_partner)
```

Comment:
Could the page text have "error" in it but not have errored?

e.g. if the description text has the word "error" would it skip here?

Reply:
The latter seems to be the case, the `in` keyword looks for keys.